src/read.c

```diff
@@ -613,8 +613,8 @@ struct FANSI_state FANSI_read_next(struct FANSI_state state) {
 
   // Normal ASCII characters
   if(chr_val >= 0x20 && chr_val < 0x7F) state = read_ascii(state);
```

Comment:
PS, is there an off-by-one here? Should this be `<= 0x7F`?

Reply:
This is intentional.  The 'del' character is treated the same way as 0x00 - 0x20 (zero width).  I don't know why the creators of ASCII threw it at the end of the list instead of the C0 section at front (I'm sure there is a good historical reason for this).